# The Blue, Green, Red are components of the RGB value
# custom led_bitfield_t
	uint8 blue_byte  # Blue Byte
	uint8 green_byte  # Green Byte
	uint8 red_byte  # Red Byte
